options.js

```diff
@@ -1,7 +1,8 @@
-import { join } from 'node:path'
 import { readFileSync } from 'node:fs'
 import eslint from 'eslint'
 
+// eslintConfig.configFile have problem reading URLs and file:///
+const configFile = new URL('./eslintrc.json', import.meta.url).toString().slice(7)
```

Comment:
Can we use `fileURLToPath` (`import { fileURLToPath } from 'node:url'`) instead of `slice`?

```suggestion
const configFile = fileURLToPath(new URL('./eslintrc.json', import.meta.url))
```

Reply:
yes